docs/images.md

```diff
@@ -71,10 +71,11 @@ Crop the image by settings both `width` and `height` attributes. The image will
 |width     |          |Resize image to specified width in pixels
 |height    |          |Crop & resize image to specified height in pixels
 |alt       |          |Alternate text for the image
-|fit 			 |`"cover"` |How to crop images. See properties below.
+|fit 			   |`"cover"` |How to crop images. See properties below.
+|position  |`"centre"`|Position of the of the visible part for 'cover' or 'contain'. (`top`, `right top`, `right`, `right bottom`, `bottom`, `left bottom`, `left`, `left top`).
```

Comment:
Can you fix the alignment of for `fit` option line? Second, thanks for adding `position` option.

Reply:
I accidentally used tab for it it seems. Should be fixed by now.